yogstation/code/modules/uplink/uplink_item.dm

```diff
@@ -70,6 +70,6 @@
 /datum/uplink_item/role_restricted/cluwneburger
 	name = "Cluwne Burger"
 	desc = "A burger infused with the tears of thousands of cluwnes infects anyone who takes a bite with a cluwnification virus which will turn them into a cluwne"
```

Comment:
Put a period after the first “cluwnes” and make a second setence starting with “infects”